pipenv/utils/indexes.py

```diff
@@ -99,8 +54,6 @@ def get_source_list(
 
 
 def parse_indexes(line, strict=False):
-    from argparse import ArgumentParser
```

Comment:
I don't agree with the linter here.
```
from argparse import ArgumentParser
```

Can be used locally inside a function if it's imported to be used in a different place.
Imports can be expensive (argparse isn't giant, but still...).
The linter rule here should be relaxed.